CI test on NB07 failed

The execution of cell 15 (counting from 0) in notebook **07 common problems** failed.

<details>
<summary>Traceback</summary >

```python-traceback
=================================== FAILURES ===================================
__________________________________ ::Cell 15 ___________________________________
Notebook cell execution failed
Cell 15: Cell execution caused an exception

Input:
hmm = msm.coarse_grain(2)

Traceback:

---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-16-7f20ceab1a9d> in <module>()
----> 1 hmm = msm.coarse_grain(2)

/opt/conda/conda-bld/pyemma_tutorials_1529581325930/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/lib/python3.6/site-packages/pyemma/msm/estimators/maximum_likelihood_msm.py in coarse_grain(self, ncoarse, method)
    806             'nstates must be an int in [2,msmobj.nstates]'
    807 
--> 808         return self.hmm(ncoarse)
    809 
    810     ################################################################################

/opt/conda/conda-bld/pyemma_tutorials_1529581325930/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/lib/python3.6/site-packages/pyemma/msm/estimators/maximum_likelihood_msm.py in hmm(self, nhidden)
    777         estimator = MaximumLikelihoodHMSM(lag=self.lagtime, nstates=nhidden, msm_init=self,
    778                                           reversible=self.is_reversible, dt_traj=self.dt_traj)
--> 779         estimator.estimate(self.discrete_trajectories_full)
    780         return estimator.model
    781 

/opt/conda/conda-bld/pyemma_tutorials_1529581325930/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/lib/python3.6/site-packages/pyemma/_base/estimator.py in estimate(self, X, **params)
    410         if params:
    411             self.set_params(**params)
--> 412         self._model = self._estimate(X)
    413         # ensure _estimate returned something
    414         assert self._model is not None

/opt/conda/conda-bld/pyemma_tutorials_1529581325930/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/lib/python3.6/site-packages/pyemma/msm/estimators/maximum_likelihood_hmsm.py in _estimate(self, dtrajs)
    260         self._dtrajs_obs = dtrajs
    261         self.set_model_params(P=transition_matrix, pobs=observation_probabilities,
--> 262                               reversible=self.reversible, dt_model=self.timestep_traj.get_scaled(self.lag))
    263 
    264         # TODO: perhaps remove connectivity and just rely on .submodel()?

/opt/conda/conda-bld/pyemma_tutorials_1529581325930/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/lib/python3.6/site-packages/pyemma/msm/models/hmsm.py in set_model_params(self, P, pobs, pi, reversible, dt_model, neig)
     95 
     96         """
---> 97         _MSM.set_model_params(self, P=P, pi=pi, reversible=reversible, dt_model=dt_model, neig=neig)
     98         # set P and derived quantities if available
     99         if pobs is not None:

/opt/conda/conda-bld/pyemma_tutorials_1529581325930/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/lib/python3.6/site-packages/pyemma/msm/models/msm.py in set_model_params(self, P, pi, reversible, dt_model, neig)
    150         self.update_model_params(P=P)
    151         # pi might be derived from P, if None was given.
--> 152         self.update_model_params(pi=pi, dt_model=dt_model, neig=neig)
    153 
    154     def __eq__(self, other):

/opt/conda/conda-bld/pyemma_tutorials_1529581325930/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/lib/python3.6/site-packages/pyemma/_base/model.py in update_model_params(self, **params)
     77         for key, value in params.items():
     78             if not hasattr(self, key):
---> 79                 setattr(self, key, value)  # set parameter for the first time.
     80             elif getattr(self, key) is None:
     81                 setattr(self, key, value)  # update because this parameter is still None.

/opt/conda/conda-bld/pyemma_tutorials_1529581325930/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/lib/python3.6/site-packages/pyemma/msm/models/msm.py in pi(self, value)
    283         if value is None and self.P is not None:
    284             from msmtools.analysis import stationary_distribution as _statdist
--> 285             value = _statdist(self.P)
    286         elif value is not None:
    287             # check sum is one

/opt/conda/conda-bld/pyemma_tutorials_1529581325930/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/lib/python3.6/site-packages/msmtools/analysis/api.py in stationary_distribution(T)
    344     # is the stationary distribution unique?
    345     if not is_connected(T, directed=False):
--> 346         raise ValueError("Input matrix is not weakly connected. "
    347                          "Therefore it has no unique stationary "
    348                          "distribution. Separate disconnected components "

ValueError: Input matrix is not weakly connected. Therefore it has no unique stationary distribution. Separate disconnected components and handle them separately
```

</details>

--- a/notebooks/07-common-problems.ipynb
+++ b/notebooks/07-common-problems.ipynb
@@ -338,7 +338,8 @@
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
-    "scrolled": true
+    "scrolled": true,
+    "skip": true
    },
    "outputs": [],
    "source": [

